src/main/java/lv/ctco/javaschool/app/control/TripStore.java

```diff
@@ -4,10 +4,12 @@
 import lv.ctco.javaschool.app.entity.domain.TripStatus;
 import lv.ctco.javaschool.auth.entity.domain.User;
 
+import javax.ejb.Stateless;
 import javax.persistence.EntityManager;
 import javax.persistence.PersistenceContext;
 import java.util.List;
 
+@Stateless
```

Comment:
You just declared this repository as a _stateless enterprise java bean_. Why? Which EJB container do you use?